### DISCUSSION TITLE
Assistance in recovering deleted documents from Phoenix Code

### DISCUSSION THREAD

**Discussion by jrsbarbara** [phcode-dev/phoenix#1614] [General]
I was working on a project in Phoenix Code using files from my desktop, and I deleted some documents from the Phoenix Code section. I believed this would only remove the files within the software; however, it also deleted them from my desktop. These documents did not go to the recycle bin and appear to be permanently erased. Could someone please assist me in recovering these critical documents?

**Comment by abose** [comment]
Closing as duplicate of https://github.com/orgs/phcode-dev/discussions/1613
Please follow tagged issue for details
